diff --git a/cjs/react-server-dom-webpack-writer.node.production.min.server.js b/cjs/react-server-dom-webpack-writer.node.production.min.server.js
index 4394bdf1cfd80330b37fff9207d2c47a26e39ac9..ffa34990d303224cd1a24549e26c8f6a5a2c7d86 100644
--- a/cjs/react-server-dom-webpack-writer.node.production.min.server.js
+++ b/cjs/react-server-dom-webpack-writer.node.production.min.server.js
@@ -8,6 +8,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 'use strict';var aa=require("util"),e=require("react"),k=null,l=0,m=!0;function p(a,b){a=a.write(b);m=m&&a}
+aa.TextEncoder = aa.TextEncoder || TextEncoder
+aa.TextDecoder = aa.TextDecoder || TextDecoder
 function q(a,b){if("string"===typeof b){if(0!==b.length)if(2048<3*b.length)0<l&&(p(a,k.subarray(0,l)),k=new Uint8Array(2048),l=0),p(a,r.encode(b));else{var d=k;0<l&&(d=k.subarray(l));d=r.encodeInto(b,d);var c=d.read;l+=d.written;c<b.length&&(p(a,k),k=new Uint8Array(2048),l=r.encodeInto(b.slice(c),k).written);2048===l&&(p(a,k),k=new Uint8Array(2048),l=0)}}else 0!==b.byteLength&&(2048<b.byteLength?(0<l&&(p(a,k.subarray(0,l)),k=new Uint8Array(2048),l=0),p(a,b)):(d=k.length-l,d<b.byteLength&&(0===d?p(a,
 k):(k.set(b.subarray(0,d),l),l+=d,p(a,k),b=b.subarray(d)),k=new Uint8Array(2048),l=0),k.set(b,l),l+=b.byteLength,2048===l&&(p(a,k),k=new Uint8Array(2048),l=0)));return m}var r=new aa.TextEncoder;function t(a){return r.encode(a)}var w=JSON.stringify,x=Symbol.for("react.module.reference"),y=Symbol.for("react.element"),ba=Symbol.for("react.fragment"),z=Symbol.for("react.provider"),ca=Symbol.for("react.forward_ref"),da=Symbol.for("react.memo"),A=Symbol.for("react.lazy"),B=Symbol.for("react.default_value");
 function C(a,b,d,c,f,g,h){this.acceptsBooleans=2===b||3===b||4===b;this.attributeName=c;this.attributeNamespace=f;this.mustUseProperty=d;this.propertyName=a;this.type=b;this.sanitizeURL=g;this.removeEmptyString=h}var D="children dangerouslySetInnerHTML defaultValue defaultChecked innerHTML suppressContentEditableWarning suppressHydrationWarning style".split(" ");D.push("innerText","textContent");D.forEach(function(a){new C(a,0,!1,a,null,!1,!1)});
