diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 844297f0982cca5fbd7ef9fb26a3f9e82eae476c..0000000000000000000000000000000000000000
diff --git a/lib/react-intl.d.ts b/lib/react-intl.d.ts
index e24bb8a41ecbf948de4c08486afb0efac36afad4..b9dec1e421f10923a27a806613d1f09ec31dec07 100644
--- a/lib/react-intl.d.ts
+++ b/lib/react-intl.d.ts
@@ -186,7 +186,7 @@ export declare interface IntlFormatters {
     formatDisplayName(value: Parameters<DisplayNames['of']>[0], opts?: FormatDisplayNameOptions): string | undefined;
 }
 
-export declare class IntlProvider extends React.PureComponent<OptionalIntlConfig, State> {
+export declare class IntlProvider extends React.PureComponent<React.PropsWithChildren<OptionalIntlConfig>, State> {
     static displayName: string;
     static defaultProps: Pick<IntlConfig, "formats" | "messages" | "timeZone" | "textComponent" | "defaultLocale" | "defaultFormats" | "onError">;
     private cache;
